src/components/01-branding/02-typography/paragraph/paragraph-default.config.yml

```diff
@@ -0,0 +1,13 @@
+label: Paragraph
+status: wip
+preview: "@sam"
+variants:
+  - name: default
+    label: Paragraph
+    context:
+      label:
+
+  - name: links
```

Comment:
Just having the name property for the link variant will suffice since no context is being added and label will be unnecessary after filename change.